constants/serverPackets.py

```diff
@@ -529,4 +529,4 @@ def invalidChatMessage(username: str) -> bytes:
     )
 
 
-popChat = packetHelper.buildPacket(packetIDs.server_popChat)
+getAttention = packetHelper.buildPacket(packetIDs.server_getAttention)
```

Comment:
we should revert my old pr which tried to optimize static packet builds to avoid the function call cuz it makes the api bad